Source/Components/Touchpoints/Types/DynamicCoverCarousel/Card/MLBusinessDynamicCoverCarouselItemView.swift

```diff
@@ -198,14 +221,46 @@ public class MLBusinessDynamicCoverCarouselItemView: UIView {
         }
         
         backgroundImageView.update(coverMedia: content.getCoverMultimedia(), cover: content.getImageHeader())
+        
+        if let topPlainImage = content.getTopPlainImage() {
+            imageProvider.getImage(key: topPlainImage) { [weak self] image in
+                self?.topPlainImageView.image = image
+            }
+
+            self.addSubview(topPlainImageView)
+            NSLayoutConstraint.activate([
```

Comment:
Tal vez estaría bueno meter todas estas logicas de layout en una función, para que no queden dentro de update.

Reply:
esto es algo que queria mostrarte porque como está hecho no está bien, esta tiene que ser la forma correcta de hacerlo cuando queres mostrar componentes dependiendo lo que viene de BE